File: Math/2652. Sum Multiples/2652. Sum Multiples.py
#Define Function
def sumOfMultiples(n):
    out = 0
    for i in range(1,n+1):
        if i % 3 == 0 or i % 5 == 0 or i % 7 == 0:
            out += i
        else:
            pass
    return out

#Run Scenarios
print(sumOfMultiples(7))
print(sumOfMultiples(10))
print(sumOfMultiples(9))
    

File: Math/2652. Sum Multiples/2652. Sum Multiples.cpp
#include <iostream>

using namespace std;

//Define Function
int sumOfMultiples(int n){
    int out = 0;
    for ( int i = 1; i < n+1; i++){
        if (i % 3 == 0){
            out += i;
        }
        else if (i % 5 == 0){
            out += i;
        }
        else if (i % 7 == 0){
            out += i;
        }
    }
    return out;

}

//Run Scenarios
int main(){
    cout << sumOfMultiples(7) << endl;
    cout << sumOfMultiples(10) << endl;
    cout << sumOfMultiples(9);
}
